<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;</string>
    </property>
    <property name="text">
     <string>MANAGER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/forgot_password1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_email">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
padding: 10px 10px;
border: 2px solid rgb(225, 225, 225);

</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="email_verify">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
color: rgb(109, 109, 109);
border: 2px solid rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/email_icon.png</normaloff>../icons/email_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>150</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string>Enter your email adress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgb(0, 190, 164);
color: white;
border-radius: 20px;
border: 0;
}
*:hover{
background-color: rgb(0, 225, 189);
}</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otp_entry">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
padding: 10px 12px;
</string>
    </property>
    <property name="placeholderText">
     <string>Enter OTP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otp_img">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/otp_icon.png</normaloff>../icons/otp_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_pass">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
padding: 10px 12px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_pass_img">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/key_icon.png</normaloff>../icons/key_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="get_otp">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: rgb(0, 190, 164);
color: white;
border-radius: 20px;
border: 0;
}
*:hover{
background-color: rgb(0, 225, 189);
}</string>
    </property>
    <property name="text">
     <string>GET OTP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eye_img">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);;
border-radius: 5px;
color: rgb(109, 109, 109)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/eye-on_icon.png</normaloff>../icons/eye-on_icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Exit Button</string>
    </property>
    <property name="statusTip">
     <string>Redirecting to Home</string>
    </property>
    <property name="styleSheet">
     <string notr="true">*{background-color: transparent;}
*:hover{background-color: rgb(220, 220, 220);
border: none;
border-radius: 7px;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/back_icon.png</normaloff>../icons/back_icon.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
